config.json.sample

```diff
@@ -0,0 +1,3 @@
+{
+    "featureManagerIsFullZoneProvisioningEnabled": false
+}
```

Comment:
@jwineman we can't write comments on `.json` files. We need to let the user figure out what settings are configurable. From this sample file as a user I'd say only `featureManagerIsFullZoneProvisioningEnabled` is configurable. 

In my opinion, we should have all the default settings which can be configurable in the sample. I've listed the items which I think should be configurable. 

```
{
    "debug": false, 
    "featureManagerIsFullZoneProvisioningEnabled": true, 
    "homePageCards": [
        "AlwaysOnlineCard",
        "IPV6Card", 
        "CacheLevelCard",
        "RailgunCard",
        "PurgeCacheCard"
    ],
    "moreSettingsCards": {
        "container.moresettings.security": [
            "SecurityLevelCard", 
            "ChallengePassageCard", 
            "BrowserIntegrityCheckCard"
        ],
        "container.moresettings.speed": [
            "MinifyCard", 
            "DevelopmentModeCard", 
            "BrowserCacheTTLCard"
        ]
    },
    "locale": "en",
}
```